api_reference.md

```diff
@@ -121,29 +126,14 @@ our [documentation](https://api-docs.midtrans.com/)
 </table>
 
 
-<a name="charge"></a>
-
-### Charge
-
-**For SNAP**
-
-```ruby
-q = Midtrans.create_snap_token(
-        transaction_details: {
-                order_id: generate_order_id,
-                gross_amount: 100000
-        })
+## Usage Example
 
-q.class # => Midtrans::Result
-q.data == {
-  status_code: "201",
-  "token": "2b3ccb6c-d0fb-499a-9d46-ef53ad51fe62",
-  "redirect_url": "https://app.sandbox.midtrans.com/snap/v2/vtweb/2b3ccb6c-d0fb-499a-9d46-ef53ad51fe62"
-}
-```
+### Create Transaction
```

Comment:
Why is every sub-section under this `Create Transaction` section level now? Only Core API & Snap create transaction should be under this right?